spec/nmt.md

```diff
@@ -126,9 +126,10 @@ As an example, the namespace proof for `nID = 0` for the NMT of Figure 1 (which
 
 ### Verification of NMT Absence Proof
 
-An NMT absence proof is deemed valid if 
-1) The verification of Merkle inclusion proof of the returned leaf is valid.  
-2) It satisfies the proof completeness as explained in the [Verification of NMT Inclusion Proof](#verification-of-nmt-inclusion-proof).
+An NMT absence proof is deemed valid if it meets the following:
+1) The minimum namespace ID of the leaf in the proof is greater than the queried `nID`. 
```

Comment:
```suggestion
1) The namespace ID of the leaf in the proof is greater than the queried `nID`. 
```

Is it safe to remove "minimum" here? The leaf doesn't have a min/max namespace and instead it has a singular namespace ID

Reply:
> The leaf doesn't have a min/max namespace and instead it has a singular namespace ID

Structurally, NMT leaves (but not their underlying data items) have two namespace IDs i.e., min and max, however, they carry identical values.